File: pay/src/main/java/com/example/pay/MainActivity.java
package com.example.pay;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Toast;

import com.alibaba.android.arouter.facade.annotation.Route;

@Route(path = "/a/aa")
public class MainActivity extends AppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toast.makeText(this, "111:" + 111, Toast.LENGTH_SHORT).show();

    }
}

File: app/src/main/java/com/example/financial/MainActivity.kt
package com.example.financial

import android.view.View
import androidx.fragment.app.Fragment
import com.alibaba.android.arouter.facade.annotation.Route
import com.example.financial.ui.home.HomeFragment
import com.example.financial.ulit.DataUlit
import com.example.frame_library.mvp.BaseActitvty
import com.example.frame_library.mvp.IPresneter
import kotlinx.android.synthetic.main.fragment_home.*

@Route(path = DataUlit.AROUTER_JUMP_MAIN)
class MainActivity : BaseActitvty<IPresneter>() ,View.OnClickListener{

    override fun bandLayoutId(): Int {
        return R.layout.activity_main
    }

    override fun initView() {
        var homeFragment = HomeFragment()
        var fragment = Fragment()
        supportFragmentManager
            .beginTransaction()
            .add(R.id.act_main_LL,homeFragment,"0")
            .add(R.id.act_main_LL,fragment,"1")
            .commit()

        supportFragmentManager
            .beginTransaction()
            .show(homeFragment)
            .hide(fragment)
            .commit()
    }

    override fun initData() {

    }

    override fun onClick(v: View?) {

    }

    override fun setPresneter(): IPresneter {
        TODO("Not yet implemented")
    }
}